LiteFarmOrg/LiteFarm
# Lf 2669-after retiring a sensor it isnt immediately removed from the farm map

To test:
1. Upload a sensor if you don't already have on
2. Click on it in the map and retire it

You should see a success message snackbar and the sensor should no longer appear on the map
